File: app/src/main/java/de/bornholdtlee/defaultproject/api/RetrofitInterface.java
package de.bornholdtlee.defaultproject.api;

import de.bornholdtlee.defaultproject.model.QuestionList;
import io.reactivex.Observable;
import retrofit2.Response;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface RetrofitInterface {

    /**
     * Beispielhafter Aufbau eines Endpoints
     */
    @GET("/2.2/questions?order=desc&sort=creation&site=stackoverflow")
    Observable<Response<QuestionList>> loadQuestions(@Query("tagged") String tags);
}

File: app/src/main/java/de/bornholdtlee/defaultproject/api/RetrofitInterface.kt
package de.bornholdtlee.defaultproject.api

import de.bornholdtlee.defaultproject.model.QuestionList
import io.reactivex.Observable
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Query

interface RetrofitInterface {

    /**
     * Beispielhafter Aufbau eines Endpoints
     */
    @GET("/2.2/questions?order=desc&sort=creation&site=stackoverflow")
    fun loadQuestions(@Query("tagged") tags: String): Observable<Response<QuestionList>>
}